cloudinit/config/schemas/schema-cloud-config-v1.json

```
@@ -308,7 +308,7 @@
           ]
         },
         "uid": {
-          "description": "The user's ID. Default is next available value.",
+          "description": "The user's ID. Default value [system default].",
```

We don't want trailing period/stop here as it gets rendered in docs as `uid: (integer/string) The user’s ID. Default value [system default]..`
```suggestion
          "description": "The user's ID. Default value [system default]",
```

---

Other occasions we could clean up to avoid the duplicated stop ".." 
```diff
--- a/cloudinit/config/schemas/schema-cloud-config-v1.json
+++ b/cloudinit/config/schemas/schema-cloud-config-v1.json
@@ -297,7 +297,7 @@
                 "string",
                 "null"
               ],
-              "description": "Sudo rule to use or false. Absence of a sudo value or ``null`` will result in no sudo rules added for this user."
+              "description": "Sudo rule to use or false. Absence of a sudo value or ``null`` will result in no sudo rules added for this user"
             },
             {
               "type": "boolean",
@@ -308,7 +308,7 @@
           ]
         },
         "uid": {
-          "description": "The user's ID. Default value [system default].",
+          "description": "The user's ID. Default value [system default]",
           "oneOf": [
             {
               "type": "integer"
@@ -1308,7 +1308,7 @@
           "properties": {
             "mode": {
               "default": "auto",
-              "description": "The utility to use for resizing. Default: ``auto``\n\nPossible options:\n\n* ``auto`` - Use any available utility\n\n* ``growpart`` - Use growpart utility\n\n* ``gpart`` - Use BSD gpart utility\n\n* ``off`` - Take no action.",
+              "description": "The utility to use for resizing. Default: ``auto``\n\nPossible options:\n\n* ``auto`` - Use any available utility\n\n* ``growpart`` - Use growpart utility\n\n* ``gpart`` - Use BSD gpart utility\n\n* ``off`` - Take no action",
               "oneOf": [
                 {
                   "enum": [
@@ -2624,7 +2624,7 @@
               "changed_description": "Use of non-boolean values for this field is deprecated."
             }
           ],
-          "description": "Sets whether or not to accept password authentication. ``true`` will enable password auth. ``false`` will disable. Default: leave the value unchanged. In order for this config to be applied, SSH may need to be restarted. On systemd systems, this restart will only happen if the SSH service has already been started. On non-systemd systems, a restart will be attempted regardless of the service state."
+          "description": "Sets whether or not to accept password authentication. ``true`` will enable password auth. ``false`` will disable. Default: leave the value unchanged. In order for this config to be applied, SSH may need to be restarted. On systemd systems, this restart will only happen if the SSH service has already been started. On non-systemd systems, a restart will be attempted regardless of the service state"
         },
         "chpasswd": {
           "type": "object",
@@ -3053,7 +3053,7 @@
       "properties": {
         "manage_etc_hosts": {
           "default": false,
-          "description": "Whether to manage ``/etc/hosts`` on the system. If ``true``, render the hosts file using ``/etc/cloud/templates/hosts.tmpl`` replacing ``$hostname`` and ``$fdqn``. If ``localhost``, append a ``127.0.1.1`` entry that resolves from FQDN and hostname every boot. Default: ``false``.",
+          "description": "Whether to manage ``/etc/hosts`` on the system. If ``true``, render the hosts file using ``/etc/cloud/templates/hosts.tmpl`` replacing ``$hostname`` and ``$fdqn``. If ``localhost``, append a ``127.0.1.1`` entry that resolves from FQDN and hostname every boot. Default: ``false``",
           "oneOf": [
             {
               "enum": [
```